app/main.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask
```

Comment:
you shouldn't have `__pycache__` files in the git repo.  remove and add to `.gitignore`